docs/modules/dataset.md

```diff
@@ -20,7 +20,9 @@ dataset = SequenceDataset(
         "dones",
     ),
     seq_length=10,                  # length-10 temporal sequences
+    frame_stack=10,                 # prepend with length-10 temporal sequence
```

Comment:
Should both seq length and frame stack be 10? I'd imagine that common use cases would be setting either one of those to 10 but not both, depending on whether using RNN or Transformer. @snasiriany can you weigh in?

Reply:
Or is this because the transformer can predict future actions as well? e.g. history of H observations to predict next T actions

Reply:
In any case, we might want to clarify the difference between seq_length and frame_stack below

Reply:
We are currently only supporting either predicting the past 10 actions or the next action. We have internal support for predicting future actions too but not in the public release yet. It doesn't hurt to have seq length be 10, that will just add overhead for data loading. I'm ok either way. Should we make seq length be 10 to make things future proof?